pom.xml

```diff
@@ -1,69 +1,71 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-    <artifactId>motech-ui-testing</artifactId>
-    <packaging>pom</packaging>
-    <version>0.27.1-SNAPSHOT</version>
+	<artifactId>motech-ui-testing</artifactId>
+	<packaging>pom</packaging>
+	<version>0.27.1-SNAPSHOT</version>
 
-    <name>MOTECH UI Functional Testing</name>
+	<name>MOTECH UI Functional Testing</name>
 
-    <modules>
-        <module>framework</module>
-        <module>tests</module>
-    </modules>
+	<modules>
+		<module>framework</module>
+		<module>tests</module>
+	</modules>
 
-    <properties>
-        <motech.version>0.27.5</motech.version>
-        <motech.dir>${user.home}/.motech</motech.dir>
-        <motech.sql.dbtype>mysql</motech.sql.dbtype>
+	<properties>
+		<motech.version>0.27.6</motech.version>
+		<motech.dir>${user.home}/.motech</motech.dir>
+		<motech.sql.dbtype>mysql</motech.sql.dbtype>
 
-        <scm.connection>scm:git:https://github.com/motech/UITestFramework</scm.connection>
-        <scm.developerConnection>scm:git:https://github.com/motech/UITestFramework</scm.developerConnection>
-        <scm.url>https://github.com/motech/UITestFramework</scm.url>
-    </properties>
+		<scm.connection>scm:git:https://github.com/motech/UITestFramework</scm.connection>
+		<scm.developerConnection>scm:git:https://github.com/motech/UITestFramework</scm.developerConnection>
+		<scm.url>https://github.com/motech/UITestFramework</scm.url>
+	</properties>
 
-    <scm>
-        <connection>${scm.connection}</connection>
-        <developerConnection>${scm.developerConnection}</developerConnection>
-        <url>${scm.url}</url>
-      <tag>HEAD</tag>
-  </scm>
+	<scm>
+		<connection>${scm.connection}</connection>
+		<developerConnection>${scm.developerConnection}</developerConnection>
+		<url>${scm.url}</url>
+		<tag>HEAD</tag>
+	</scm>
 
-    <parent>
-        <groupId>org.motechproject</groupId>
-        <artifactId>motech</artifactId>
-        <version>0.27.5</version>
-    </parent>
+	<parent>
+		<groupId>org.motechproject</groupId>
+		<artifactId>motech</artifactId>
+		<version>0.27.5</version>
```

Comment:
I have added and I got this message:

[ERROR]   The project org.motechproject:motech-ui-testing:0.27.1-SNAPSHOT (C:\Users\User_1\Dropbox\sandbox\motech\ruben\UITestFramework\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM: Failure to find org.motechproject:motech:pom:${motech.version} in http://nexus.motechproject.org/content/repositories/public was cached in the local repository, resolution will not be reattempted until the update interval of motech-repo has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 34, column 10 -> [Help 2]
org.apache.maven.model.resolution.UnresolvableModelException: Failure to find org.motechproject:motech:pom:${motech.version} in http://nexus.motechproject.org/content/repositories/public was cached in the local repository, resolution will not be reattempted until the update interval of motech-repo has elapsed or updates are forced

I have all the modules , motech in the same level so all are in the same folder and the error still is there (http://goo.gl/L8I39H)

I would love to use the dynamic reference , perhaps is needed another change in the motech pom , is it not working right now.

Just the 0.27.6 works, so it seems something on configuration ?